.github/workflows/operator-verify.yaml

```diff
@@ -17,3 +22,30 @@ jobs:
       - uses: actions/checkout@v4
       - name: run test
         run: make -C components/operator test
+
+  # pre-serverless-operator-verify, post-serverless-operator-verify
+  serverless-operator-verify:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false
```

Comment:
This job should be run only for PR ready for review because we need built image.